### ISSUE TITLE
CLI `cpx` copies a file to wrong place

### ISSUE THREAD

**Issue by sanemat** [mysticatea/cpx#11]
CLI `cpx` copies a file to wrong place.

```
$ cpx a.txt dist

expected
./a.txt
./dist/a.txt

actual
./a.txt
./adisttxt

$ cpx ./a.txt dist

expected and actual
./a.txt
./dist/a.txt
```

This is my environment.
```
cpx v1.2.0
Mac OS X 10.9.5
node.js v0.12.7
```

**Comment by mysticatea** [comment]
Thank you for the report.

I will check it next evening, since I'm busy today...

**Comment by mysticatea** [comment]
I'm sorry, I will do until week end :tired_face:

**Comment by mysticatea** [comment]
Sorry for my late response.

I could reproduce it.
I'm working on this now.

**Comment by mysticatea** [comment]
@sanemat Thank you for the report.

I fixed and released `v1.2.1`.
Please try it.

**Comment by sanemat** [comment]
@mysticatea Great work! Thanks. :surfer:
